packages/nextjs/pages/projects/[projectId].tsx

```diff
@@ -1,88 +1,147 @@
+import YourBallot from "~~/components/op/projects/YourBallot";
+import IncludedProjects from "~~/components/project/IncludedProjects";
+import ProjectHeader from "~~/components/project/ProjectHeader";
+import Sidebar from "~~/components/shared/Sidebar";
 import dbConnect from "~~/lib/dbConnect";
 import Project, { ProjectDocument } from "~~/models/Project";
-import { notification } from "~~/utils/scaffold-eth";
 
 interface Props {
   projects: ProjectDocument[];
 }
 
 const ProjectDetail: NextPage<Props> = ({ projects }) => {
-  const [isLiked, setIsLiked] = React.useState(false);
-  const [isAdded, setIsAdded] = useState(false);
-  const { state, dispatch } = useBallot();
+  const { isDisconnected } = useAccount();
+  const [wallet, setWallet] = useState<boolean | false>(false);
+  // TODO: Add these entries to the database for each document and pull from there
+  const impactArray = [
+    {
+      id: 1,
+      name: "Tx Supported",
+      amount: "1.39M",
+    },
+    {
+      id: 2,
+      name: "total tx amount",
+      amount: "$500M",
+    },
+    {
+      id: 3,
+      name: "Total Downloads",
+      amount: "500",
+    },
+  ];
 
   useEffect(() => {
-    if (!state) return;
-    setIsAdded(false);
-    const isAddedToBallot = () => {
-      state.projects.forEach(x => {
-        if (x.id === projects[0]._id) setIsAdded(true);
-      });
-    };
-    isAddedToBallot();
-  }, [projects, state]);
-
-  const addProjectToBallot = () => {
-    const _name = projects[0].name as string;
-    dispatch({
-      type: "ADD_PROJECT",
-      project: {
-        id: projects[0]._id,
-        name: _name,
-        allocation: 0,
-      },
-    });
-    notification.success("Added to ballot");
-  };
+    setWallet(isDisconnected);
+  }, [isDisconnected]);
 
   return (
-    <div className=" mx-auto px-12 mt-12 grid gap-12">
+    <div className="mx-auto px-12 mt-12 grid lg:grid-cols-[350px,1fr] gap-4 pb-16">
+      {!wallet ? <YourBallot /> : <Sidebar />}
       <div className="">
-        <div className="grid mb-3 sm:grid-flow-col items-center">
-          <h3 className="text-2xl font-bold">{projects[0].name}</h3>
-          <div className="grid grid-flow-col gap-4 w-fit sm:w-full sm:justify-end relative">
-            <button
-              onClick={() => setIsLiked(!isLiked)}
-              className=" flex items-center gap-1 rounded-xl p-4 border-[1px] border-[#CBD5E0]"
-            >
-              <span>12</span>
-              {isLiked ? (
-                <HeartIcon className="w-6 h-6  text-[#68778D]" />
+        <ProjectHeader projects={projects} />
+
+        <div className=" mt-8 mb-8 border-t border-gray-200 w-full"></div>
+        <div className="">
+          <section id="about">
+            <h3 className="font-medium text-[#68778D] text-xl leading-7 uppercase">👨‍💻 About</h3>
+            <p className="font-normal leading-6  text-based">{projects[0] ? projects[0].description : ""}</p>
```

Comment:
I think that we need to assign **project** by selecting projectId from projects and then use **project** here.

Reply:
Good call. I agree. Right now, for whatever reason we are passing in a projects array but only with one entry that is the selected project. Not sure why we are dong that but I went ahead and simplified it by defining "project" and using that everywhere instead of "projects[0]".